src/main/java/seedu/address/logic/commands/ImportCommand.java

```diff
@@ -72,17 +73,36 @@ public ImportCommand(Path filePath) {
         this.filePath = filePath;
     }
 
+    /**
+     * Generates a report of the import process.
+     */
+    private String generateReport(int[] importResults) {
+        int successfulImports = importResults[0];
+        int unsuccessfulImports = importResults[1];
+        return String.format(MESSAGE_IMPORT_SUCCESS, filePath.toString()) + "\n"
+            + String.format("Successful imports: %d\n", successfulImports)
+            + String.format("Unsuccessful imports: %d\n", unsuccessfulImports)
+            + errorMsgs;
+    }
+
+    private void generateErrorReport(Exception e) {
+        errorMsgs += e.getMessage() + "\n";
```

Comment:
Might wanna consider how you should change the AddCommandParser to ensure you highlight missing parameters.